mlkem/config.h

```diff
@@ -235,6 +235,41 @@
  *****************************************************************************/
 /* #define MLK_FIPS202X4_CUSTOM_HEADER "SOME_FILE.h" */
 
+/******************************************************************************
+ * Name:        MLK_DESTRUCT_INTERMEDIATES
+ *
+ * Description: If this option is set, functions are instrumented to zeroize
+ *              intermediate stack buffers prior to returning.
+ *
+ *              WARNING:
+ *              This feature reduces the likelihood of secrets leaking
+ *              on the stack, but does not eliminate it!
+ *              The C standard makes no guarantee about where a compiler
+ *              allocates structures and whether/where it makes copies of
+ *              them. Also, in addition to entire structures, there may also
+ *              be potentially exploitable leakage of individual values on the
+ *stack.
+ *
+ *              If you need bullet-proof zeroization of the stack, you need to
+ *              consider additional measures atop of what this feature provides.
+ *
+ *              The default implementation uses SecureZeroMemory on Windows
+ *              and a memset + compiler barrier otherwise. If you want to
+ *replace this default implementation, set MLK_USE_CT_ZEROIZE_NATIVE and provide
+ *a definition for `ct_zeroize_native(void *ptr, size_t len);`.
+ *
+ *****************************************************************************/
+#define MLK_DESTRUCT_INTERMEDIATES
+/* #define MLK_USE_CT_ZEROIZE_NATIVE
+   #if !defined(__ASSEMBLER__)
+   #include "sys.h"
+   static MLK_INLINE void ct_zeroize_native(void *ptr, size_t len)
```

Comment:
If I uncomment this and add my own implementation - I get a "
```
mlkem/config.h:266:56: error: unknown type name 'size_t'
  266 |    static MLK_INLINE void ct_zeroize_native(void *ptr, size_t len)
 ```
 
 If I add a `#include <stddef.h>` it works fine

Reply:
done